manifests/qpid_client.pp

```diff
@@ -39,16 +40,31 @@
 
   if $deploy {
 
+    file { $certs::pulp_pki_dir:
+      ensure => directory,
+      owner  => 'root',
+      group  => 'apache',
```

Comment:
Could you add a class parameter for this? it's fine to hardcode the default but I've found that it can be very useful to not code these values inside the body.